src: mail: Add 'additional_emails' configuration

davidbtadokoro opened the pull request:
`kw mail` already has a configuration to automatically block emails as recipients named 'blocked_emails'. An opposite configuration, to automatically add emails as recipients doesn't exist, though.

In this context, add the configuration 'additional_emails' to `kw mail` to automatically add pre-determined recipients.

Commit:
src: mail: Add configurations for default recipients

`kw mail` already has a configuration to automatically block emails as
recipients named 'blocked_emails'. An opposite configuration, to
automatically add emails as recipients doesn't exist, though.

In this context, add the configurations 'default_to_recipients' and
'default_cc_recipients' to automatically add To: and CC: recipients,
respectively.

Closes: #908

Signed-off-by: David Tadokoro <davidbtadokoro@usp.br>

rodrigosiqueira commented on etc/mail.config:
```diff
@@ -6,3 +6,7 @@ send_opts=--annotate --cover-letter --no-chain-reply-to --thread
+
+# You can add emails to always be included as recipients. Emails should be
+# separated by comma
+additional_emails=
```
I don't think this field expresses the idea behind it really well. Maybe we can try something like `demails_always_included` (it is still a bad name, it is just an example)...

We also should have one for `to` and another one for `cc`.

rodrigosiqueira reviewed (changes requested):
In general, it lgtm. Remember to add a test and reference @melissawen issue in the commit message. Also, expand the man documentation.

davidbtadokoro commented:
Hey @rodrigosiqueira, thanks for the feedback!
I updated the PR addressing your comments.
In respect to the configurations name, I used `default_to_recipients` and `default_cc_recipients`. What do you think?

melissawen commented:
names sounds good to me.

rodrigosiqueira commented:
Overall, it lgtm. @melissawen I'll wait for your okay for this change before I merge it.

melissawen commented:
Just tested and it works as expected. I noticed that when I add the same e-mail address to `default_to_recipients` and `default_cc_recipients` it doesn't repeat and the list of e-mails in `To:` is prioritized. All good, you can add my t-b:
melissa.srw \<at\> gmail.com
